env0/resource_template_test.go

```diff
@@ -57,6 +57,7 @@ func TestUnitTemplateResource(t *testing.T) {
 			},
 		},
 		Type:               "terragrunt",
+		TerragruntVersion:  "0.35.1",
```

Comment:
now the tests are working better. (They were supposed to fail without terragrunt_version, but didn't)